src/dmd/expression.d

```diff
@@ -72,10 +72,12 @@ import dmd.typesem;
 import dmd.visitor;
 
 enum LOGSEMANTIC = false;
+
 void emplaceExp(T : Expression, Args...)(void* p, Args args)
 {
-    scope tmp = new T(args);
-    memcpy(p, cast(void*)tmp, __traits(classInstanceSize, T));
+    const init = typeid(T).initializer;
```

Comment:
Wasn't this superseded by `__traits(initSymbol)`?

Reply:
`__traits(initSymbol)` isn't available in the bootstrap compiler (2.079)

Reply:
Ah, of course

Reply:
Also, bootstrap mips64el with initSymbol to ensure that it doesn't regress under the new feature either.

Reply:
Ha! initSymbol didn't land until 2.099, you're going to have to wait until at least 2.120 because I'd allow that. ;-)